shub/log.py

```diff
@@ -36,13 +36,33 @@
 SHORT_HELP = "Fetch log from Scrapy Cloud"
 
 
+class LogIterFunc(object):
```

Comment:
Hey @kalessin Sorry for the huge delay :/

I would really rather avoid this class. I believe an "initial `startafter`" feature is very much generic enough to fit into the `job_resource_iter` core function. We may after all want to use it for `requests` or `items` too at some point (dunno if that's supported by the JobQ API).